### ISSUE TITLE
Finetuning configuration

### ISSUE THREAD

**Issue by yuellong** [dingdingcai/SC6D-pose#1]
What is the parameter setting for fine-tuning? In particular, what learning rate did you use?

**Comment by dingdingcai** [comment]
The pre-trained models are finetuned for additional 30 epochs with the cosine annealing learning rate from 1e-4 to 1e-5.

**Comment by yuellong** [comment]
Thanks for the info. I was still unable to reproduce the finetuning results. Anything important to take care of other than learning rate, e.g., balancing the real / synthetic examples?
